README.md

```diff
@@ -39,42 +38,110 @@ python -m venv noisepy
 Marine Denolle (mdenolle@uw.edu).
 
-#### Peer-Reviewed Publications Referenced
-Seats, K. J., Jesse F. L., and German A. P. "Improved ambient noise correlation functions using Welch′ s method." _Geophysical Journal International_ 188, no. 2 (2012): 513-523.
-*Jiang, C. and Denolle, M. "NoisePy: a new high-performance python tool for seismic ambient noise seismology." _Seismological Research Letter_ 91, no. 3 (2020): 1853–1866..
-** Yuan, C., Bryan, J. T., and Denolle, M. "Numerical comparison of time-, frequency- and wavelet-domain methods for coda wave interferometry." _Geophysical Journal International_ 226, no. 2 (2021): 828-846.
+## Use this reference when publishing on your work with noisepy
+
+Main code:
+* Jiang, C. and Denolle, M. [NoisePy: a new high-performance python tool for seismic ambient noise seismology.](https://doi.org/10.1785/0220190364) _Seismological Research Letter_ 91, no. 3 (2020): 1853–1866. https://doi.org/10.1785/0220190364
+
+Algorithms used:
+* (data pre-processing) Seats, K. J., Jesse F. L., and German A. P. [Improved ambient noise correlation functions using Welch′ s method.](https://doi.org/10.1111/j.1365-246X.2011.05263.x) _Geophysical Journal International_ 188, no. 2 (2012): 513-523. https://doi.org/10.1111/j.1365-246X.2011.05263.x
+
+* (dv/v in wavelet domain) Yuan, C., Bryan, J. T., and Denolle, M. [Numerical comparison of time-, frequency- and wavelet-domain methods for coda wave interferometry.](https://doi.org/10.1093/gji/ggab140) _Geophysical Journal International_ 226, no. 2 (2021): 828-846. https://doi.org/10.1093/gji/ggab140
+
+* (optimal stacking) Yang X, Bryan J, Okubo K, Jiang C, Clements T, Denolle MA. [Optimal stacking of noise cross-correlation functions/](https://doi.org/10.1093/gji/ggac410) _Geophysical Journal International_. 2023 Mar;232(3):1600-18. https://doi.org/10.1093/gji/ggac410
+
+
+
+## Parameters to configure
+
+The parameters of the workflow are saved into an object called ``ConfigParameter``. The parameters
+
+ *   dt: float = 0.05  # TODO: dt should be 1/sampling rate
+ *   start_date: str = ""  # TODO: can we make this datetime?
+ *   end_date: str = ""
+ *   samp_freq: float = 20  # TODO: change this samp_freq for the obspy "sampling_rate"
+*  cc_len: float = 1800.0  # basic unit of data length for fft (sec)
+ *   # download params.
+ *   # Targeted region/station information: only needed when down_list is False
+ *  lamin: float = 31
+ *   lamax: float = 36
+ *   lomin: float = -122
+ *   lomax: float = -115
+ *   down_list = False  # download stations from a pre-compiled list or no
+ *   net_list = ["CI"]  # network list
+    # pre-processing parameters
+ *   step: float = 450.0  # overlapping between each cc_len (sec)
+ *  freqmin: float = 0.05
+ *   freqmax: float = 2.0
+ *   freq_norm: str = "rma"  # choose between "rma" for a soft whitenning or "no" for no whitening
+ *   #  TODO: change "no" for "None", and add "one_bit" as an option
```

Comment:
This is markdown, so lines that start with `#` become headers. You can preview this in VS Code ( right-click on the file and Open Preview. I also suggest removing all `# TODO` comments from here.

<img width="530" alt="image" src="https://user-images.githubusercontent.com/10456546/236870148-3ceb7695-cd44-4b37-b448-e7efbcd4e340.png">